curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript-libraries-and-frameworks/6734e88cc46e6dc679420040.md

```diff
@@ -149,15 +144,15 @@ Which directory contains the starting code that is found in the `App.jsx` file?
 
 ### --feedback--
 
-Review the end of the video where this was discussed.
+Review the section where this was discussed.
```

Comment:
and here 

Let's make this more specific. like "reviewing the end section" or something like that. Apply to all feedbacks